### PR TITLE
Help document

### PR DESCRIPTION
**Pull request by kyoh86**
This plugin is so convenient.
So I try proposing a document.

### CODE REVIEW

File: doc/tig-explorer.txt [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+TigOpenCurrentFile                                       *:TigOpenCurrentFile*
+    Open tig with current file
+
+TigOpenProjectRootDir                                 *:TigOpenProjectRootDir*
+    Open tig with Project root path
+
+TigGrep                                                             *:TigGrep*
+    Open tig grep.
+    And you can pass fraze like `:TigGrep (fraze)` to open with the fraze.
```
Comment (iberianpig) [inline doc/tig-explorer.txt]:
```suggestion
    And you can pass PATTERN like `:TigGrep (PATTERN)` to grep with the PATTERN.
```

**Review by iberianpig — CHANGES_REQUESTED** [review]
Thanks for creating a pull request.
Could you modify the following sentences?

File: doc/tig-explorer.txt (hunk 1 above)
Comment (kyoh86) [reply doc/tig-explorer.txt]: thanks

**Comment by kyoh86** [comment]
@iberianpig Thank you for your review.
I fixed words!

**Comment by iberianpig** [comment]
Thanks for your contribution.👍

**Pull request merged**